raspberry/bzzz/sensors/gps_lea6s.py

```diff
@@ -0,0 +1,117 @@
+from math import degrees
+import serial
+import time
+from threading import Thread
+import numpy as np
+
+
+def deg_min_sec_to_decimal(
+    #dms, dir="N"
+    degrees, minutes, direction
+    ):
+    """
+    Convert degrees-minutes-seconds from GNGLL to decimal
+
+    :param dms: degrees-minutes-seconds (must be positive)
+    :param dir: direction (can be one of 'N', 'W', 'E', 'S')
+    """
+    
+    decimal_degrees = degrees + minutes / 60
+    if direction in ['S', 'W']:  # need to be denoted with a "-" value -> are negative
+        decimal_degrees *= -1
+    return decimal_degrees
+
+class GpsLea6SReader:
```

Comment:
should we change this to ZED-F9R instead of LEA6S

Reply:
This code is an NMEA parser. It's independent of the module. We should change it to something more generic.